frontend/glideinFrontendElement.py

```diff
@@ -955,6 +957,38 @@ def iterate_one(self):
 
         return
 
+    def getScheddList(self):
+        # Get the original list from config
+        schedd_list = self.elementDescript.merged_data.get("JobSchedds", [])
+
+        if schedd_list != ["ALL"]:
+            # If the frontend admin specified a scheduler list use it.
+            return schedd_list
+
+        # logSupport.log.info("Getting list of schedulers from collector since 'ALL' has been used") # Too verbose. Keeping it around until production deployment
+        # If the admin specified ALL then query the condor collector to get the list of schedulers
+        res = glideinFrontendLib.getCondorStatusSchedds([None], constraint=None, format_list=[])
+
+        if res and None in res:
+            schedds = list(res[None].fetchStored().keys())
+
+            # Cache the result with timestamp
+            self.schedd_cache["data"] = schedds
+            self.schedd_cache["timestamp"] = time.time()
```

Comment:
This is more a comment than a change request.
Seems more a backup value than a cache. It is used only if the query failed and the timestamp is not used. Or is it used somewhere else I'm missing?

Reply:
Argh, I messed things up and I forgot to include an if :(

Reply:
Added the missing code to adderess cache usage here: https://github.com/glideinWMS/glideinwms/pull/532/commits/2e2154388197a5fbfb7b749b9e8f07b835e4aa7b